setup.py

```diff
@@ -3,7 +3,12 @@
 setup(
     name='quickscreen',
     version='0.1',
-    packages=find_packages(exclude=['tests*']),
+    packages=find_packages(exclude=[
+        'tests*',
+         "data", 
+         "example.ipynb",
+         "description.md"
+         ]),
```

Comment:
and I guess the README as well as it's github repo specific?

Reply:
I'm looking at this right now, I may not need to list anything that isn't a directory...